vic/CMakeLists.txt

```diff
@@ -0,0 +1,207 @@
+cmake_minimum_required(VERSION 2.8.0)
+project(vic)
+# ============================================================================
+# Settings. 
+# ----------------------------------------------------------------------------
+
+set(OTHER_INCLUDE_PATH  your_include_path)  # Add other include path you want.
+set(OTHER_LIBRARY_PATH  your_library_path)  # Add other library path you want.
+set(MPI_COMPILER        mpicc)              # Compiler of MPI.
+
+# ============================================================================
+
+# Cmake configurations options. ==============================================
+option(CLASSIC      "Compile VIC Classic Driver"    OFF)
+option(IMAGE        "Compile VIC Image Driver"      ON)
+option(SHARED_LIB   "With Vic shared library"       OFF)
+option(INSTALL      "Install VIC to system path"    OFF)
+# ============================================================================
+
+set(CMAKE_INSTALL_PREFIX /usr/local/vic) # Path to install VIC shared library.
```

Comment:
most users will not have write permissions here.  I would suggest defaulting to either the driver directory (e.g. `VIC/vic/drivers/image/`) or `VIC/bin/`.